app/graphql/types/union/mapping_item.rb

```diff
@@ -0,0 +1,5 @@
+class Types::Union::MappingItem < Types::Union::Base
+  description 'Objects which are Mappable'
+
+  possible_types Types::Anime, Types::Manga
```

Comment:
Quick spitup from our database:

| Type | Count |
|----|---:|
| Anime | 55920 |
| Category | 132 |
| Character | 105669 |
| Episode | 792 |
| Manga | 53298 |
| Person | 35053 |
| Producer | 1326 |

Reply:
Thank you, I will add this as a concern `has_many :mappings, as: 'item', dependent: :destroy` and start adding to the Types that are missing (I know for certain character doesn't respond to `character.mappings`

Reply:
@NuckChorris can you also run a query to get me all `external_sites` we have?

Reply:
| external_site | count |
|---|---:|
| anidb | 10194 |
| anilist | 8994 |
| animenewsnetwork | 114 |
| aozora | 12183 |
| hulu | 910 |
| imdb/episodes | 792 |
| kitsu/genres | 51 |
| mangaupdates | 2210 |
| myanimelist/anime | 15501 |
| myanimelist/character | 105669 |
| myanimelist/genres/anime | 40 |
| myanimelist/genres/manga | 41 |
| myanimelist/manga | 50680 |
| myanimelist/people | 35017 |
| myanimelist/person | 36 |
| myanimelist/producer | 1326 |
| mydramalist | 2 |
| thetvdb | 2965 |
| thetvdb/season | 621 |
| thetvdb/series | 2157 |
| trakt | 2687 |

Reply:
@NuckChorris what resources are getting mapped by `myanimelist` and `thetvdb` without suffixes?

Reply:
anilist was mainly from zorro I think. thetvdb is from our background job (will verify). Nothing for myanimelist?

Reply:
Anilist is garbage data, there's no bare MAL in the list (you might be seeing mydramalist)

Dunno about TVDB

Reply:
I wrote `myanimelist` for `anilist` for some reason - then checked the list again and didn't see `anilist` in my message so added `anilist` "again". Oops